configure.ac

```diff
@@ -1455,13 +1455,15 @@ AC_MSG_NOTICE([Linux Netfilter support requested: ${enable_linux_netfilter:=auto
 dnl Look for libnetfilter_conntrack options (needed for QOS netfilter marking)
 SQUID_AUTO_LIB(netfilter-conntrack,[Netfilter conntrack],[LIBNETFILTER_CONNTRACK])
 SQUID_CHECK_LIB_WORKS(netfilter-conntrack,[
-  LDFLAGS="$LIBNETFILTER_CONNTRACK_PATH $LDFLAGS"
+  SQUID_STATE_SAVE(squid_netfilter_conntrack_state)
+  PKG_CHECK_MODULES([LIBNETFILTER_CONNTRACK],[libnetfilter_conntrack],[:],[:])
   CPPFLAGS="$LIBNETFILTER_CONNTRACK_CFLAGS $CPPFLAGS"
-  AC_SEARCH_LIBS([nfct_query],[netfilter_conntrack],[LIBNETFILTER_CONNTRACK_LIBS="-lnetfilter_conntrack"])
+  LIBS="$LIBNETFILTER_CONNTRACK_PATH $LIBNETFILTER_CONNTRACK_LIBS $LIBS"
```

Comment:
besides consistency, what is the reason to touch LIBS here? the following tests only check for headers

Reply:
consistency is the main reason, to ensure that these checks still work properly when the LIBS are set instead of LDFLAGS.